POSTINSTALL-README.md

```diff
@@ -117,3 +124,22 @@ The custom rule allows the group`diskadmin` to do the actions for unlocking and
 ## Instruction set optimizations for hardened_malloc
 
 Please see the description for release [v2.2.0](https://github.com/secureblue/secureblue/releases/tag/v2.2.0)
+
+## Add a SELinux confined user
+[SELinux confined users](https://fedoraproject.org/wiki/SELinux/ConfinedUsers) are a project to secure not only the core system, but also the user accounts with [SELinux](https://docs.fedoraproject.org/en-US/quick-docs/selinux-getting-started/).
+
+At the current state, unlike on Android, a user account and all it's processes are not protected by SELinux on Fedora.
+
+As this may cause breakages, [follow these guides](https://fedoraproject.org/wiki/SELinux/ConfinedUsers#Assign_a_SELinux_user_to_an_existing_Linux_user) on how to add a new user account and confine it.
```

Comment:
We need to incorporate these steps into the guide for user provisioning instead of just linking to the fedora docs. it should be one seamless set of steps

Reply:
okay, I wanted to avoid duplication and make it more maintainable that way. The link is also directly to the steps needed, the other step (creating the user) is included there.

So this should not be in the FAQ but stay here?